lib/gruff/base.rb

```diff
@@ -1169,7 +1174,7 @@ def label(value, increment)
       else
         label = sprintf('%0i', value)
       end
-
+      label = "#{y_label_prefix}#{label}" unless y_label_prefix.empty?
```

Comment:
can do `label.prepend(y_label_prefix.to_s)` which will also handle when `y_label_prefix` is nil
actually given the default is an empty string, you dont need the `to_s`